provisioning/cli-dist/resources/github-action-production-role.json

```diff
@@ -41,9 +41,12 @@
                 "StringEquals": {
                   "token.actions.githubusercontent.com:aud": "sts.amazonaws.com"
                 },
-                "StringLike": {
+                "ForAnyValue:StringLike": {
                   "token.actions.githubusercontent.com:sub": {
-                    "Fn::Sub": "repo:${GitHubOrganization}/${RepositoryName}:ref:refs/heads/main"
+                    "Fn::Sub": [
+                      "repo:${GitHubOrganization}/${RepositoryName}:ref:refs/heads/main",
+                      "repo:${GitHubOrganization}/${RepositoryName}:ref:refs/tags/v*"
+                    ]
```

Comment:
Po uprave pipeline release spadne:
![image](https://user-images.githubusercontent.com/19371734/201615517-148bf257-2111-4343-8a58-9a34c3e0c337.png)

Pretoze sa S3 provisining spusta uz aj vramci CLI release, z tagu.

Preto tu musim upravit tuto rolu.

Tento JSON subor sa pouziva v `create-github-production-role.sh` scripte, ktory sa myslim nevola nikde v CI.
https://github.com/keboola/keboola-as-code/blob/michaljurecko-ba-33-fix-role/provisioning/cli-dist/scripts/create-github-production-role.sh

Mam ten script spustit rucne? Odkial zoberiem ENVs v nom? Napr. `GITHUB_OIDC_PROVIDER_ARN`.
Poradis prosim @JakubMatejka ?

Reply:
Aha, uz som to nasiel tu: https://github.com/keboola/keboola-as-code/blob/main/docs/RELEASE.md#3-create-aws-roles-for-github-actions-to-production-environment 

Ok. Tak len pre kontrolu, ci to mozem spustit a aktualizuje sa to ako ocakavam.

Reply:
(Alebo to bude lepsie upravit rucne v AWS Console?)

Reply:
Jo jo, mělo by. Musíš si akorát smaozřejmě nastavit cli pro přítup do toho produkčního accountu.

Reply:
To je taky možnost. 🙂

Reply:
jj to bude v pohode, len mi nebolo jasne ci to vytvori novu rolu, alebo aktualizuje existujucu. dik, idem to spustit.

Reply:
Novou určitě ne. Otázka je jestli si CloudFormation poradí s updatem, ale při nejhorším spadne. 😄

Reply:
poradi si s updatem https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html#cfn-iam-role-assumerolepolicydocument
<img width="1025" alt="image" src="https://user-images.githubusercontent.com/903531/201627488-79812983-19e3-440b-8c40-86d8793c4c18.png">

Reply:
preslo to, vdaka, hodnoty sa nezmenili:
![image](https://user-images.githubusercontent.com/19371734/201627851-d73261e7-393d-439f-8b61-7b80138c2762.png)